src/Classes/PassiveTree.lua

```diff
@@ -49,44 +49,52 @@ local function getFile(URL)
 	return #page > 0 and page
 end
 
-local PassiveTreeClass = newClass("PassiveTree", function(self, treeVersion)
-	self.treeVersion = treeVersion
-	local versionNum = treeVersions[treeVersion].num
-
-	self.legion = LoadModule("Data/TimelessJewelData/LegionPassives")
-	self.tattoo = LoadModule("Data/TattooPassives")
-
-	MakeDir("TreeData")
-
-	ConPrintf("Loading passive tree data for version '%s'...", treeVersions[treeVersion].display)
-	local treeText
-	local treeFile = io.open("TreeData/"..treeVersion.."/tree.lua", "r")
-	if treeFile then
-		treeText = treeFile:read("*a")
-		treeFile:close()
+-- Retrieve the file at the given Path
+local function getFile(Path, altPath)
+	local text
+	local File = io.open("TreeData/"..Path..".lua", "r")
```

Comment:
nit: Can we keep variable names lowercased if they are just an instance of a class?
```suggestion
	local file = io.open("TreeData/"..Path..".lua", "r")
```